### DISCUSSION TITLE
Redux from the first Cogent3 workshop

### DISCUSSION THREAD

**Discussion by GavinHuttley** [cogent3/cogent3#1642] [General]
Let's collate the feedback from attendee's here and filter that for actions which we can create on the issues

**Comment by GavinHuttley** [comment]
- We should have signs to help people find the room!
- We should have catering for snacks and tea
- gloss stickers, die cut

**Comment by KatherineCaley** [comment]
- We should start with an intro to cogent3 section. Without this, no problem will feel surmountable for a beginner. This should give participants practice in the basic style of cogent3 programming and using the docs. Should be very straightforward. 
- All key takeaways from the "experimental design" section of the workshop need to be motivated with hands-on examples. 
  - e.g. , sometimes exons do not satisfy the asymptotic assumption of the methods being used. Can we come up with an example of a method that is sensitive to the length of exons? Get participants to follow along with an example that shows this. Use this as a launching pad to talk about other considerations in experimental design. 
  - What other experimental design considerations can be motivated with a "lesson"?

**Comment by YapengLang** [comment]
- People want clear navigation of available apps in the [User Guides](https://cogent3.org/doc/index.html). 
  It is important to guide users on when to use specific apps and where to locate available apps in the documentation. For example, in section 4, when users seek an app to load Phylip files sequentially, they may not get enough information using the search bar. In such cases, they anticipate finding relevant demos in the Cookbook or Tutorials, but these examples currently focus only on manipulating single files.

**Comment by khiron** [comment]
One problem I ran into with the github wiki that you probably didn't notice unless you used git to access the hidden repo behind the wiki (https://github.com/cogent3/Cogent3Workshop.wiki.git), is that github's sanitizers don't strip out ':' characters.  

So you can certainly title a markdown page "Section 2: Getting Data [mac only emoji published as a blank box on windows]💾"

But if someone is using the git repo to, say, push images and videos and a zip of the devcontainer up to the repo to be hosted without resorting to the hack of sticking things in issues ... then when you try to sync with the git repo git refuses to sync with some cryptic errors.  Which is something I ran into around lunch of the day of the workshop.

Initially I thought the problem could be the emojis ... but they were fine.  Turns out that a colon in the filename of the md file is accepted by githubs wiki server, but no self respecting file system would allow a colon in a filename - so git on your clone will suddenly not work.  So you might have seen that the names of all the content sections changed slightly - that's why.
